SpatialGDK/Source/SpatialGDK/Public/Utils/LatencyManager.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+
+class USpatialNetConnection;
+class USpatialNetDriver;
+
+class ULatencyManager
+{
+public:
+	ULatencyManager(USpatialNetConnection* InConnection);
+
+	void Enable(Worker_EntityId InPlayerControllerEntity);
+	void Disable();
+
+private:
+	void SendPingOrPong(Worker_ComponentId ComponentId);
+
+	Worker_EntityId PlayerControllerEntity;
+	float LastPingSent;
+
+	USpatialNetConnection* NetConnection;
```

Comment:
Are we sure this is the right class? Isn't this created per player, when this class is meant to be for all players/

Reply:
This should be here, one latency manager per player with Ping being a replicated property on PlayerState that's available to other players